src/control/control.h

```diff
@@ -137,6 +137,22 @@ class ControlDoublePrivate : public QObject {
         return m_key;
     }
 
+    void enable() {
```

Comment:
I am not in favour of this change, because it puts extra complexity to ALL our COs. 
Once this is merged, the code and mappings has to deal with this additional state. 
What are alternative solutions?

Reply:
I agree. How about a `[Skin], has_menubar` CO that skins can set to 1?